numpy/ma/extras.py

```diff
@@ -258,7 +258,8 @@ def getdoc(self):
         doc = getattr(npfunc, '__doc__', None)
         if doc:
             sig = self.__name__ + ma.get_object_signature(npfunc)
-            locdoc = "Notes\n-----\nThe function is applied to both the _data"\
+            locdoc = "\n\n        Notes\n        -----\n        "\
+                     "The function is applied to both the _data"\
                      " and the _mask, if any."
```

Comment:
Perhaps instead we should use `inspect.cleandoc(doc)` before concatenating?

Reply:
@eric-wieser Thanks for reviewing. I think that's right, so I fixed the code.